Bottom: 4d2fc79bbb4378d7a78a84e622ec4d03b4dc27a4
Top:    bfa0530b6784382d53e320b779d5de2e70153367
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-05-27 19:37:58 +0300

Refresh of test_pmalloc

---

diff --git a/mm/Makefile b/mm/Makefile
index 42610ce8f77b..545254985211 100644
--- a/mm/Makefile
+++ b/mm/Makefile
@@ -66,7 +66,7 @@ obj-$(CONFIG_SPARSEMEM_VMEMMAP) += sparse-vmemmap.o
 obj-$(CONFIG_SLOB) += slob.o
 obj-$(CONFIG_MMU_NOTIFIER) += mmu_notifier.o
 obj-$(CONFIG_PROTECTABLE_MEMORY) += pmalloc.o
-obj-$(CONFIG_TEST_PROTECTABLE_MEMORY) += test_pmalloc.o
+obj-$(CONFIG_DEBUG_PROTECTABLE_MEMORY_TEST) += test_pmalloc.o
 obj-$(CONFIG_KSM) += ksm.o
 obj-$(CONFIG_PAGE_POISONING) += page_poison.o
 obj-$(CONFIG_SLAB) += slab.o
